Dockerfile

```diff
@@ -36,6 +36,9 @@ WORKDIR $APPDIR
 # Set the locale, this is required for some of the Python packages
 ENV LC_ALL C.UTF-8
 
+# Ensure pip is up to date
+RUN pip3 install --upgrade pip
```

Comment:
Can you please add `--no-cache` to avoid polluting docker image with pip wheel caches.